backend/data/src/main/kotlin/io/tolgee/service/notification/NotificationService.kt

```diff
@@ -33,15 +35,19 @@ class NotificationService(
       .toInt()
 
   @Transactional
-  fun save(notification: Notification) {
-    emailNotificationsService.sendEmailNotification(notification)
-    notificationRepository.save(notification)
-    applicationEventPublisher.publishEvent(
-      OnNotificationsChangedForUser(
-        notification.user.id,
-        notification,
-      ),
-    )
+  fun notify(notification: Notification) {
+    if (notificationSettingService.getSettingValue(notification, NotificationChannel.EMAIL)) {
+      emailNotificationsService.sendEmailNotification(notification)
```

Comment:
Prevent sending e-mail notifications to deleted users and disabled users (please test it)

Reply:
I would do this in a separate PR, I made a note about it.